### PR TITLE
Adjust KEB plans schemas

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Adjusted KEB plans schemas

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/plans_validator_test.go [hunk 1]
```diff
@@ -13,20 +13,10 @@ func TestNewPlansSchemaValidatorErrors(t *testing.T) {
 		inputJSON    string
 		expErr       string
 	}{
-		"missing name, not valid components list": {
-			againstPlans: []string{GCPPlanID, AzurePlanID},
-			inputJSON:    `{"components": ["wrong component name"]}`,
-			expErr:       `(root): name is required, components.0: components.0 must be one of the following: "kiali", "tracing"`,
-		},
-		"missing name, not valid machine type": {
```
Comment (jasiu001) [inline components/kyma-environment-broker/internal/broker/plans_validator_test.go:21]: should we remove this test?

Comment (polskikiel) [reply components/kyma-environment-broker/internal/broker/plans_validator_test.go:21]: we shouldn't - I have reverted tests for regions and machine types 👍

**Review by jasiu001 — APPROVED** [review]

**Pull request merged**